apm-sniffer/optional-reporter-plugins/kafka-reporter-plugin/src/main/java/org/apache/skywalking/apm/agent/core/kafka/KafkaTraceSegmentServiceClient.java

```diff
@@ -72,13 +119,7 @@ public void afterFinished(final TraceSegment traceSegment) {
             LOGGER.debug("Trace[TraceId={}] is ignored.", traceSegment.getTraceSegmentId());
```

Comment:
previous version is better？
```
    @Override
    public void afterFinished(TraceSegment traceSegment) {
        if (traceSegment.isIgnore()) {
            LOGGER.debug("Trace[TraceId={}] is ignored.", traceSegment.getTraceSegmentId());
            return;
        }
        if (!carrier.produce(traceSegment)) {
            if (LOGGER.isDebugEnable()) {
                LOGGER.debug("One trace segment has been abandoned, cause by buffer is full.");
            }
        }
    }
```

Reply:
@nileblack If you want to enhance more, we are welcome you to submit pull request.